server/src/main/kotlin/fi/oph/kitu/dev/MockLoginController.kt

```diff
@@ -50,7 +51,12 @@ class MockLoginController(
                 oid = "",
                 strongAuth = false,
                 kayttajaTyyppi = "VIRKAILIJA",
-                authorities = listOf(),
+                authorities =
+                    listOf(
+                        SimpleGrantedAuthority("APP_KIELITUTKINTOREKISTERI"),
+                        SimpleGrantedAuthority("APP_KIELITUTKINTOREKISTERI_READ"),
+                        SimpleGrantedAuthority("APP_KIELITUTKINTOREKISTERI_READ_1.2.246.562.10.00000000001"),
```

Comment:
Näihin kun lisää `ROLE_` eteen, niin mock loginin (ja playwright-testien) pitäis alkaa toimia.